README.md

```diff
@@ -21,6 +23,28 @@ docker run -it -v /var/lib/kubelet/device-plugins:/var/lib/kubelet/device-plugin
 kubectl create -f nvidia-device-plugin.yml
 ```
 
+### Local Env
+
+You `MUST` set environment variable `LIBRARY_PATH` for static linking `cudart_static` library which required [here](https://github.com/NVIDIA/nvidia-docker/blob/master/src/cuda/bindings.go#L5), as show below (Ubuntu 16.04):
```

Comment:
This is too complicated. You can just assume that the `/usr/local/cuda` symlink exists.
And then you can do:
```bash
C_INCLUDE_PATH=/usr/local/cuda/include LIBRARY_PATH=/usr/local/cuda/lib64 go build
```

Reply:
@flx42 Thanks, good idea!